testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/suite/suites/Suite7NonGeneric.java

```diff
@@ -49,6 +50,9 @@ public List<SuiteTestRun> getTestRuns(EnvironmentConfig config)
                 testOnEnvironment(EnvSinglenodeSparkHive.class)
                         .withGroups("configured_features", "hive_spark")
                         .build(),
+                testOnEnvironment(EnvSinglenodeSparkHiveNoStatsFallback.class)
```

Comment:
What is `EnvSinglenodeSparkHive` missing in comparison to `EnvSinglenodeSparkHiveNoStatsFallback` ?
I'm trying to figure out why we are introducing yet another environment here.

Reply:
Different `hive.conf`